peerdid/peer_did_resolver.py

```diff
@@ -0,0 +1,11 @@
+from peerdid.types import JSON
+
+
+class PeerDIDResolver:
+    @staticmethod
+    def resolve(self, peer_did: str, version_id=None) -> JSON:
+        pass
+
```

Comment:
Another thing we may check with Daniel.

When we create a new peer DID (method 0 or method 2), it produces an implicit DID DOC under the hood. 
For example, for did:peer:0 (which is similar to did:key) this is something like [Example did:key DID DOC](https://github.com/digitalbazaar/did-method-key-js#example-did-document)
For did:peer:2 this is something like [Example did:peer:2 DID DOC](https://identity.foundation/peer-did-method-spec/index.html#multi-key-creation).

So, the questions are
1) Do we need to return the corresponding DID Doc when we create a new peer DID, so that it can be used to call `save` later
2) When saving a did doc for the given peer did, should we check that the did doc matches the peer did 
3) It looks like it's sufficient to store just peer DID itself for :0 and :2 peer DIDs, as it can be converted to a did doc. So, should that fact be represented in an API (have a `save` version where only a peer did can be provided)?